plugins/eddn.py

```diff
@@ -226,18 +245,40 @@ def send(self, cmdr: str, msg: Mapping[str, Any]) -> None:
                 logger.trace_if('plugin.eddn', "EDDN is still objecting to empty commodities data")
                 return  # We want to silence warnings otherwise
 
+            if r.status_code == 413:
+                self._log_response(r, header_msg='Got a 413 while POSTing data', sent_data_len=str(len(encoded)))
```

Comment:
Can we log at least the `$schemaRef` for the message, and if it's `/journal/` the `event` value ?